src/config.rs

```diff
@@ -51,3 +52,19 @@ pub fn zo_maxage() -> Result<Rank> {
         None => Ok(1000.0),
     }
 }
+
+pub fn zo_fzf_extra_args() -> Result<Vec<String>> {
```

Comment:
Here, too, it would be great if you could arrange the functions in alphabetical order, since there is no other way to logically arrange the functions, and would prove helpful as the codebase grows!